client/components/search/index.jsx

```diff
@@ -38,7 +38,8 @@ module.exports = React.createClass( {
 		onKeyDown: React.PropTypes.func,
 		disableAutocorrect: React.PropTypes.bool,
 		onBlur: React.PropTypes.func,
-		searching: React.PropTypes.bool
+		searching: React.PropTypes.bool,
+		forceCloseButton: React.PropTypes.bool
```

Comment:
@matias - I've looked at the `Search` component and I do think there's a case for this new prop. 

`/design` and `/sites` do not display the close button when the input is empty.

`/posts` has a close button when the input is empty, but I think only because it starts off pinned.

I don't think there's currently a way to force the close button to appear when using an unpinned `Search`.